packages/flutter/lib/src/gestures/events.dart

```diff
@@ -636,16 +629,14 @@ class PointerExitEvent extends PointerEvent {
   /// Creates an exit event from a [PointerEvent].
   ///
   /// This is used by the [MouseTracker] to synthesize exit events.
-  PointerExitEvent.fromMouseEvent(PointerEvent event) : super(
+  PointerExitEvent.fromMouseEvent(PointerEvent event) : this(
```

Comment:
please add tests that verify the behaviour you change by using `this` instead of `super`

Reply:
Added via 1f384d58134bf25fc3ebd203e498e68a8967631a